prep-release.sh

```diff
@@ -24,19 +24,19 @@ make generate
 
 message="build(flux): prepare Flux release for $version"
 
-git commit -am "$message"
+git commit --allow-empty -am "$message"
 git push
 
-if ! command -v hub &> /dev/null
+if ! command -v gh &> /dev/null
```

Comment:
`gh` has replaced `hub` so I've updated the script, rather than spend seconds creating a PR by hand.